app/scripts/controllers/network/network-controller.test.ts

```diff
@@ -7998,3 +8169,40 @@ async function waitForLookupNetworkToComplete({
     count: numberOfNetworkDetailsChanges,
   });
 }
+
+/**
+ * Returns whether two places in different state objects have different values.
+ *
+ * @param currentState - The current state object.
+ * @param prevState - The previous state object.
+ * @param propertyPath - A property path within both objects.
+ * @returns True or false, depending on the result.
+ */
+function isStateChangeInteresting(
+  currentState: Record<string | symbol | number, unknown>,
```

Comment:
Nit: The built-in type `PropertyKey` can be used anyplace `string | symbol | number` is used

Reply:
Oh, that's neat! Yeah, that's a lot better.